src/components/__tests__/Comments.test.js

```diff
@@ -138,4 +138,27 @@ describe('ReplyList tests', () => {
     });
 });
 
+describe('LikeDislikeComments tests', () => {
+    const props = {
+        id: 1,
+        likeCount: 3,
+        dislikeCount:1
+    };
+    const wrapper = shallow(< LikeDislikeComment {...props}/>);
+
+    it('renders without crashing', () => {
```

Comment:
You might want to check this test out @pbkabali. I mean it's like the test is not testing any thing

Reply:
@FahdJamy let me check on this. Your feedback is much appreciated. thanks